tests/nightly/cpu/test_convai2_cpu.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import unittest
+import parlai.utils.testing as testing_utils
+
+
+"""
+This module ensures that the baseline models for convai2 produce the
+correct ~~validation~~ results.
```

Comment:
what is with the ~~

Reply:
idk, this was just there. All i did was rename this file.